tms/scalar/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""Scalar entrypoint."""
+
+
+import asyncio
+import logging
+from typing import Any
+
+import htcondor  # type: ignore[import-untyped]
+from rest_tools.client import RestClient
+
+from .. import utils
+from ..config import ENV
+from . import starter, stopper
+
+LOGGER = logging.getLogger(__name__)
+
+
+async def next_to_start(ewms_rc: RestClient) -> dict[str, Any]:
+    """Get the next taskforce requested for this collector + schedd."""
+    return await ewms_rc.request(  # type: ignore[no-any-return]
+        "GET",
+        "/tms/taskforce/pending",
+        {"collector": ENV.COLLECTOR, "schedd": ENV.SCHEDD},
+    )
```

Comment:
When there is no task to start, does this raise an exception or return none?

Reply:
it will return an empty dict. I added a comment on that